library/core/src/char/convert.rs

```diff
@@ -199,6 +199,8 @@ impl const From<u8> for char {
 }
 
 /// An error which can be returned when parsing a char.
+///
+/// This `struct` is created when using the [`char::from_str`](char#tymethod.from_str) method.
```

Comment:
I think it's still not ready:

```
warning: unresolved link to `char::from_str`
 --> foo.rs:1:7
  |
1 | /// [`char::from_str`]
  |       ^^^^^^^^^^^^^^ the builtin type `char` has no associated item named `from_str`
```

We need https://github.com/rust-lang/rust/issues/90703 to be fixed first.